### ISSUE TITLE
Add smoke tests for replicator in CI

### ISSUE THREAD

**Issue by elpaso** [hotosm/underpass#54]
Currently we have no tests for the replicator application, all existing tests are library API tests.

In order to avoid merging changes that break the main replicator application we should add a set of smoke tests that check that replicator basic functionality is not broken, for example:

1. it does not crash when a DB connection is wrong/goes down
2. can perform the basic operations (monitoring, user sync, osm replication etc.)

**Comment by robsavoye** [comment]
Correct. Currently the few test cases are unit tests, not integration tests. Good unit tests are more important for now that testing replicator, but yes, it should have some sanity tests. Actual integration tests not using replicator would also be nice. Replicator is basically just the thing that processes command line arguments and starts the threads that do the real work.

**Comment by emi420** [comment]
I've added integration tests for raw data that covers the replication process. For DB and network connectivity testing I've created two new issues.
